Source/WebGPU/WebGPU/Buffer.h

```diff
@@ -155,12 +155,13 @@ class Buffer : public WGPUBufferImpl, public RefCounted<Buffer>, public CanMakeW
 
 } // namespace WebGPU
 
+// FIXME: rdar://137074301
```

Comment:
Not really too important, but should we leave a WebKit bug link too and not just the radar number?